ansible/playbooks/rhel.yml

```diff
@@ -170,6 +171,8 @@
     #######################
     # Additional Packages #
     #######################
+    #       ccache        #
```

Comment:
??? bad copy paste?

Reply:
I can add the extra line of hashes if you want.  I just added this as a section header.